domainservice/src/test/kotlin/ua/marchenko/artauction/auction/service/kafka/AuctionCreatedEventConsumerTest.kt

```diff
@@ -0,0 +1,107 @@
+package ua.marchenko.artauction.auction.service.kafka
+
+import artwork.random
+import auction.random
+import ch.qos.logback.classic.Logger
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.core.read.ListAppender
+import java.time.Duration
+import org.awaitility.Awaitility.await
+import kotlin.test.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.ActiveProfiles
+import reactor.kafka.receiver.KafkaReceiver
+import ua.marchenko.artauction.artwork.model.MongoArtwork
+import ua.marchenko.artauction.artwork.service.ArtworkService
+import ua.marchenko.artauction.auction.controller.dto.CreateAuctionRequest
+import ua.marchenko.artauction.auction.service.AuctionService
+import ua.marchenko.artauction.common.AbstractBaseIntegrationTest
+import ua.marchenko.artauction.common.kafka.common.createBasicKafkaConsumer
+import ua.marchenko.artauction.user.model.MongoUser
+import ua.marchenko.artauction.user.service.UserService
+import ua.marchenko.internal.KafkaTopic
+import user.random
+
+@SpringBootTest
+@ActiveProfiles("test")
+@Import(AuctionCreatedEventConsumerTest.AuctionCreatedEventConsumerTestConfiguration::class)
```

Comment:
AbstractBaseIntegrationTest already has @SpringBootTest and @ActiveProfiles("test")